spec/parity/backup_spec.rb

```diff
@@ -51,7 +51,7 @@ def database_fixture
   end
 
   def database_fixture_path
-    File.join(File.dirname(__FILE__), 'fixtures', 'database.yml')
+    File.join(File.dirname(__FILE__), '..', 'fixtures', 'database.yml')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.